controller/src/main/java/org/dswarm/controller/resources/ExtendedBasicDMPResource.java

```diff
@@ -51,6 +64,49 @@ public ExtendedBasicDMPResource(final Class<POJOCLASS> pojoClassArg, final Provi
 	}
 
 	/**
+	 * This endpoint returns an object of the type of the POJO class as JSON representation for the provided object uuid.
+	 * The format of the object might either be a full or an abbreviated short variant.
+	 *
+	 * @param uuid an object uuid
+	 * @param format an enum that specifies which format to use
+	 * @return a JSON representation of an object of the type of the POJO class
+	 */
+	@Timed
+	public Response getObject(
+			   /* @PathParam("uuid") */ final String uuid,
+			/* @QueryParam("format") */ final POJOFormat format) throws DMPControllerException {
```

Comment:
The Jersey annotations on DataModel etc. are using @DefaultValue("full") so that a missing parameter defaults to full. For programmatic access, there are still the old getObject(String) method(s).